### ISSUE TITLE
build with dnstap support

### ISSUE THREAD

**Issue by dmachard** [MatthewVance/unbound-docker#82]
Hi,

Many thanks for this works.
Could you add the **dnstap** support with the following configure option  "--enable-dnstap" in the docker file by default ?

Denis

**Comment by MatthewVance** [comment]
Hi Dennis, feel free to submit a pull request for this feature addition. If you do, please also update the Readme with instructions. Thanks for your support and contribution.

**Comment by dmachard** [comment]
Hi. Any chance to merge the PR #83 for the next unbound release 0.15.0  ?

**Comment by MatthewVance** [comment]
Yes. I’ve been meaning to do it for latest. I got behind and missed merging it with the earlier release.
